src/components/Settings/Notifications/NotificationsEmail.tsx

```diff
@@ -160,38 +159,40 @@ const NotificationsEmail: React.FC = () => {
         return (
           <>
             <Alert
-              title={intl.formatMessage(messages.emailNotificationTypesAlert)}
+              title={
+                <>
+                  <p className="mb-2">
+                    {intl.formatMessage(
+                      messages.emailNotificationTypesAlertDescription,
+                      {
+                        strong: function strong(msg) {
+                          return (
+                            <strong className="font-normal text-indigo-100">
+                              {msg}
+                            </strong>
+                          );
+                        },
+                      }
+                    )}
+                  </p>
+                  <p>
+                    {intl.formatMessage(
+                      messages.emailNotificationTypesAlertDescriptionPt2,
+                      {
+                        strong: function strong(msg) {
+                          return (
+                            <strong className="font-normal text-indigo-100">
+                              {msg}
+                            </strong>
+                          );
+                        },
+                      }
+                    )}
+                  </p>
+                </>
+              }
```

Comment:
I think this makes more sense as a child of Alert instead of all in the title? Title isn't required.

Reply:
I'll share a screenshot of it in a bit, but since all the alerts are bolded (I literally changed all of them), it looks better as the title so that it matches the other notification agent alerts.

Reply:
![image](https://user-images.githubusercontent.com/52870424/112340158-b08dd400-8c96-11eb-89ce-c2f06d2fb292.png)
![image](https://user-images.githubusercontent.com/52870424/112340184-b5528800-8c96-11eb-8fcb-5b98c857125f.png)

Reply:
Alternatively, we could make all of the alerts set the message/body instead of the title?  Let me know what you think is better.  I sort of like that the alerts are a heavier font weight, makes them easier to read on the brightly-colored background.